doc/api/cli.md

```diff
@@ -1140,6 +1141,32 @@ directory provided as an argument. Coverage is output as an array of
 }
 ```
 
+Source Map data is output as an object, where the keys of the object represent
+the files source maps were extracted from, and the values include the raw
+source-map URL (in the key `url`) and the parsed Source Map V3 information (
+in the key `data`).
```

Comment:
Where is the data coming from? Fetched by the third-party? Is it what `process._getSourceMapCache` is for?

Reply:
I apologize, `process._getSourceMapCache` was left over from a first pass at refactoring the implementation to use `inspector_profiler.cc`.

This data is output when source-map information is output, using the private method `sourceMapCacheToObject` -- which exists only to facilitate `inspector_profiler.cc` outputting the source-maps.